### PR TITLE
[Breakpad] fix cmake include install

### PR DESCRIPTION
**Pull request by jiayuehua**
**Describe the pull request**

- #### What does your PR fix?  
  Fixes breakpad port not install include dir

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  <linux>, <No>

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `yes`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  <Yes>

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/breakpad/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "breakpad",
   "version-string": "2020-09-14",
```
Comment (JonLiu1993) [inline ports/breakpad/vcpkg.json:3]:
```suggestion
  "version-date": "2020-09-14",
```

File: ports/breakpad/portfile.cmake [hunk 2]
```diff
@@ -29,9 +29,14 @@ file(
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
```
Comment (JonLiu1993) [inline ports/breakpad/portfile.cmake:31]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/breakpad/portfile.cmake [hunk 3]
```diff
@@ -28,10 +28,14 @@ file(
 
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+    OPTIONS
+    -DINSTALL_HEADERS=ON
```
Comment (JonLiu1993) [inline ports/breakpad/portfile.cmake:32]:
```suggestion
        -DINSTALL_HEADERS=ON
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@jiayuehua ,Thanks for your pr, have you tested the usage?

**Comment by jiayuehua** [comment]
> @jiayuehua ,Thanks for your pr, have you tested the usage?

Yes, I have tested.

File: ports/breakpad/portfile.cmake (hunk 3 above)
Comment (JonLiu1993) [reply ports/breakpad/portfile.cmake:32]: @jiayuehua ,After completing this suggestion, we can prepare to merge

**Review by JonLiu1993 — APPROVED** [review]

File: ports/breakpad/portfile.cmake (hunk 2 above)
Comment (BillyONeal) [reply ports/breakpad/portfile.cmake]: This was wrong, `PREFER_NINJA` is only removed with `vcpkg_cmake_configure`, not `vcpkg_configure_cmake`. @JonLiu1993 Can you take point to use the modern helpers here?

**Comment by BillyONeal** [comment]
(Alternately, I'll look at doing that modernization when I am back in the office on Monday, December 27)

File: ports/breakpad/portfile.cmake (hunk 2 above)
Comment (JonLiu1993) [reply ports/breakpad/portfile.cmake]: @BillyONeal ,Thanks for pointing out this! I will modernize it

**Comment by JonLiu1993** [comment]
> (Alternately, I'll look at doing that modernization when I am back in the office on Monday, December 27)

@BillyONeal ,I have helped users complete the function modernization modification

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the fix and modernization!

**Pull request merged**